yml/Desktop_App/vtso.yml

```diff
@@ -0,0 +1,29 @@
+---
+Name: Microsoft Visual Studio Tools for Office (VSTO)
+Description: VSTO is a functionality available in Visual Studio which allows the user to craft a .NET Office Add-in to be shipped with an office document.
+Author: 'Abhijeet Kumar'
+Created: 2023-02-10
+Usage:
+  - Steps: Create a VTSO document (Word, Powerpoint, etc) embedded with a DLL file written in .NET and send it to any user. Once the user opens the documents and approves the "Add-In" installation, code will be executed.
+    Description: Steps to exploitation using VTSO
+    Usecase: Initial Access to the system
+    Category: Phishing
+    Privileges: User interaction/User
+    Limitations: A user with valid privileges to approve Add-in installation. 
+    MitreID: T1204.002
+  - Steps: Add another "Add-In" to this document and include the code for any of the persistence techniques. After successful installation, persistence will be achieved.
```

Comment:
Modifications recommended:
Utilize a different VSTO payload "Add-In" and include the code for any of the persistence techniques. After successful installation, persistence will be achieved. For more information, see "VSTO-POC" in references.

Reply:
Done